cm_channels/cm_soundChannels.py

```diff
@@ -108,89 +107,71 @@ def calculate(self):
                 changex = math.atan2(relative[2], relative[1])/math.pi
                 self.store[emitterid] = {"rz": changez,
                                          "rx": changex,
-                                         "distProp": dist/(val)}
-
-        # Octree implementation
-        """O = bpy.context.scene.objects
-        userDim = self.sim.agents[self.userid].dimensions
-
-        if self.octree is None:
-            bss = []  # List of bounding spheres
-            for emitterid, val in self.emitterDict.items():
-                emitDim = self.sim.agents[emitterid].dimensions
-
-                dim = (val + emitDim[a] + userDim[a] for a in range(3))
-
-                bss.append(ot.boundingSphereFromBPY(O[emitterid], dim))
-
-            self.octree = ot.createOctree(bss)
-
-        ag = O[self.userid]
-
-        collisions = self.octree.checkPoint(ag.location.to_tuple())
-
-        for emitterid in collisions:
-            if emitterid == self.userid:
-                continue
+                                         "distProp": dist/val}
```

Comment:
You're absolutely right about all the commented sections in this file. With git we can get rid of all this. It was mostly left because when the new code was introduced it replaced something that was already very reliable (just a bit slow) and so I wanted a fallback but then forgot to get rid of it at a later point. It also highlights our need to make some proper testing for this project.